src/toxcore/dht/packet.rs

```diff
@@ -140,6 +140,14 @@ doesn't arrive for 122 seconds the DHT node removes peer from kbucket and marks
 it as offline if the peer is known friend.
 
 https://zetok.github.io/tox-spec/#dht-packet
+
+Length  | Content
+------- | -------------------------
+`1`     | `0x00`
+`32`    | Public Key
+`24`    | Nonce
+`9`     | Payload
```

Comment:
Encrypted Payload, where Payload is .... link to payload struct

Reply:
@NamsooCho have a look at https://github.com/tox-rs/tox/pull/50/files#diff-bd6ee0853157076924e3716122a19209R318